Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -543,6 +543,7 @@ class StoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
         } catch {
             expect(error).to(matchError(ErrorCode.ineligibleError))
         }
+        self.logger.verifyMessageWasNotLogged("API request started")
```

Comment:
You might have to clear the logs before calling the method because this contains the entire test lifetime, including `setUp`

Reply:
Fixed!